minique/api.py

```diff
@@ -84,3 +113,47 @@ def cancel_job(
             p.execute()
         return True
     return False
+
+
+def _define_and_store_job(
```

Comment:
Can you make this a kwarg-only function?

Reply:
will kwargify

Reply:
dunno if you wanted only the default-having arguments kwarg-onlied but made the whole functions all kwargs-only :test_tube:

Reply:
Whole function yes! In fact, this function shouldn't necessarily even have defaults, since the higher-level APIs have them and keeping them in sync could be a chore, but it's fine!

Reply:
> Whole function yes! In fact, this function shouldn't necessarily even have defaults, since the higher-level APIs have them and keeping them in sync could be a chore, but it's fine!

true, probably worth removing the defaults when touched next